nilearn/image/image.py

```diff
@@ -563,6 +563,8 @@ def index_img(imgs, index):
      (91, 109, 91)
     """
     imgs = check_niimg_4d(imgs)
+    if hasattr(index, 'values') and hasattr(index, 'iloc'):
```

Comment:
why the second test for 'iloc' ?

Reply:
Just to be extra careful with duck-typing. A dictionary, for instance, has a "values" method, and is not a dataframe.